src/api/app/presenters/obs_factory/staging_project_presenter.rb

```diff
@@ -61,7 +61,7 @@ def classified_requests
         reviews[r[:request]] << r
       end
       requests.each do |req|
-        r = { number: req.number, package: req.package }
+        r = { number: req.number, package: req.bs_request_actions.first.target_package }
```

Comment:
Could you use something more explainable than `r`?

Reply:
I didn't introduce this so it's a bit out of scope for this PR. But the change is small enough. I am going to add another commit in a moment